apps/server/src/controllers/Product.ts

```diff
@@ -13,6 +13,7 @@ const getAll = async (req: Request, res: Response) => {
 };
 
 const create = async (req: Request, res: Response) => {
+  console.log('Hola');
```

Comment:
please remove this console log, thanks :)

Reply:
Done